kgitb/routes.py

```diff
@@ -7,58 +7,58 @@
 
 
 def check_pr(commits_url, comments_url, status_url):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a docstring to this function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)